tutorial/lesson-1/README.md

```diff
@@ -71,32 +84,60 @@ The View runs in the client or in the normal case, a browser. The WABuR
 reference implemenation provides HTML and JavaScript using Ruby code as well
 as JavaScript libraries.
 
-Two files are needed for this lesson. A Ruby file for the UI and an HTML index
-file.
+All files necessary are generated by creating a project directory or creating
+a project in GitHub and cloning it. Then `cd` to the project
+directory. Assuming the project was called `blog` execute the following
+commands.
 
-[tell me more...](MORE.md#implementation)
+```
+$ mkdir blog
+$ cd blog
+$ wabur init Entry
+```
```

Comment:
please add reference to `wabur new` as well:
```
Or to start from scratch, locally, use `wabur new` mode. Simply run the following to create a project
directory at './blog' and initialize controllers for type Entry:

    wabur new Entry --base blog

```

Reply:
done